examples/salesforce_sink_example.py

```
@@ -28,380 +28,391 @@
-        
+
         # Create sample data for custom object
-        streaming_df = spark.readStream \
-            .format("rate") \
-            .option("rowsPerSecond", 1) \
-            .load()
-        
+        streaming_df = spark.readStream.format("rate").option("rowsPerSecond", 1).load()
+
         # Transform for custom object (example: Product__c)
         custom_data = streaming_df.select(
```

Ruff (0.12.2)

Local variable `custom_data` is assigned to but never used

Remove assignment to unused variable `custom_data`

(F841)